src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuckModule.java

```diff
@@ -80,61 +171,61 @@ public void postProcess(File file) {
 
     public final void updateMetadata(Path outputFile) {
         try {
-            NetcdfFileWriter nc = NetcdfFileWriter.openExisting(outputFile.toAbsolutePath().toString());
+            String location = outputFile.toAbsolutePath().toString();
+            NetcdfFileWriter nc = NetcdfFileWriter.openExisting(location);
+            GridDataset gridDs = GridDataset.open(location);
 
             nc.setRedefineMode(true);
             for (Attribute newAttr : getGlobalAttributesToUpdate(nc.getNetcdfFile())) {
```

Comment:
Need to cater for collection specific metadata update such as that currently performed for SRS?